spawns.lua

```diff
@@ -125,18 +125,65 @@ NorthGeorgiaFARPTransportSpawns = {
     ["SE"] = {Spawner("SE FARP HELO"),nil, SEFARPLogiSpawn},
     ["MK"] = {Spawner("MK FARP HELO"), nil, MaykopLogiSpawn}
 }
+
+-- Tells a tanker (or other aircraft for that matter) to orbit between two points at a set altitude
+
+BladderPos = {}
+BladderPos.hMaxFlightAlt	= 5486	-- meters [18 000']: Don't let the aircraft fly higher than this as Hogs won't be able to refuel. TODO: Make overrideable
+BladderPos.vSpeed			= 160		-- m/s [300kts]: Default speed of the unit
+
+function BladderPos.InitOnGroup(planeGroup, vSpeed)
```

Comment:
Can we give `InitOnGroup` a better name? It might help once the above is clarified.